app/frontend/src/pages/page_helpers.ts

```
@@ -24,37 +32,262 @@ export async function setupStore(
+    for await (const chunk of readNDJSONStream(response.body)) {
+        if (chunk as Chunk) {
+            let shouldUpdate = false;
+
+            switch (chunk.type) {
+                case "C":
+                    buffer += chunk.message as string;
+                    shouldUpdate = true;
+                    break;
+                case "I":
+                    const info = chunk.message as ChunkInfo;
+                    streamed_tokens = info.streamedtokens;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)